esp-hal-common/src/uart.rs

```diff
@@ -1423,26 +1436,36 @@ mod asynch {
             }
         }
 
-        fn event_bit_is_clear(&self) -> bool {
+        fn event_bit_is_clear(&self) -> EnumSet<RxEvent> {
```

Comment:
the function's name is now probably somewhat misleading

Reply:
Function name is updated now